user_docs/en/changes.t2t

```diff
@@ -7,6 +7,15 @@ What's New in NVDA
 
 == New Features ==
 - In Windows 11, NVDA will announce alerts from voice typing and suggested actions including the top suggestion when copying data such as phone numbers to the clipboard (Windows 11 2022 Update and later). (#16009, @josephsl)
+- New input gestures:
+  - Added gestures to jump first, last, forward and backward the values through the synth settings ring. (#13768, #16095, @rmcpantoja)

```

Comment:
Don't we use "command" instead of "gesture" here?

E.g. we say "A command with no assigned gesture". The command is "jump forward setting value in the ring" and the gesture is NVDA+control+pageUp.
```suggestion
  - Added commands to jump first, last, forward and backward through the synth settings ring. (#13768, #16095, @rmcpantoja)

```